core/src/main/java/org/openstack4j/openstack/telemetry/internal/BaseTelemetryAodhServices.java

```diff
@@ -0,0 +1,36 @@
+package org.openstack4j.openstack.telemetry.internal;
+
+import com.google.common.base.Function;
+import com.google.common.collect.Lists;
+import org.openstack4j.api.types.ServiceType;
+import org.openstack4j.openstack.internal.BaseOpenStackService;
+
+import java.util.Collections;
+import java.util.List;
+
+public class BaseTelemetryAodhServices extends BaseOpenStackService {
+
+	protected BaseTelemetryAodhServices() {
+		super(ServiceType.TELEMETRY_AODH, EndpointFunction.instance);
+	}
+	
+
+	private static class EndpointFunction implements Function<String, String> {
```

Comment:
yeah,OpenStack services can get the endpoint from the service catalog.but openstack4j is a tool for  developers.users can get service information  from osclient.BaseTelemetryAodhServices is responsible for the related work.